### ISSUE TITLE
[Bug][Scheduler] Using SchedulerProportionalViewItem with vertical grouping breaks the layout of the slots

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#2309]
## I'm submitting a...
* Bug report

## Current behavior
When the scheduler has vertical orientation for the groups and a SchedulerProportionalViewItem is used, the slots height is not correctly calculated if there are multiple events in a slot. The issue is not observed without the SchedulerProportionalViewItem

## Expected behavior
The same behavior should be observed with the default and with the SchedulerProportionalViewItem

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/react-mqs3dr?file=app%2Fmain.jsx
(you can check the expected behavior by removing the viewItem)

Package versions:
8.0.0

**Comment by pvmeijel** [comment]
Hello,

Could you please let me know when we can expect a fix? We have a client that is using the scheduler, and this bug prevents the rollout of our solution.

Could you let me know what we could expect, so that we can communicate with the client accordingly?

**Comment by kdikov82** [comment]
The SchedulerProportionalViewItem is missing the call of the useRowSync (as in the SchedulerViewItem):
```
    useRowSync({
        element: schedulerElement,
        selector: '.k-resource-row',
        horizontalAttribute: 'data-depth-index',
        verticalAttribute: 'data-resource-index',
        applyTo: '.k-resource-cell',
        syncHeight: props.isLast
    });
```

Adding the call of the userRowSync will sync the height

**Comment by PaskalevStoyan** [comment]
Fix is provided and available in `v8.2.0-develop.4`. It will be available in the next official release.
